lua/telescope/actions.lua

```diff
@@ -165,6 +165,15 @@ actions.set_command_line = function(prompt_bufnr)
   vim.cmd(entry.value)
 end
 
+actions.run_builtin = function(prompt_bufnr)
+  local entry = actions.get_selected_entry(prompt_bufnr)
+
+  actions.close(prompt_bufnr)
+  vim.cmd [[startinsert]]
+
+  require('telescope.builtin')[entry.value.text]()
```

Comment:
I thought this would be entry.text, not entry.value.text

Reply:
Either one works on my machine. I'll change to to entry.text then